ls1intum/Artemis
# `Development`: Remove outdated database column in post table

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

### Motivation and Context


With Metis, we switched from having `votes` to using dedicated emoji reaction (`+`) to express agreement and vote for a post.
The datamodel (`Post.java`) was adapted earlier in #4336, we can now safely remove the according database column `votes`.

### Description

This PR introduces a database migration, deletes a translation that is not required anymore as well as some - outdated - parts in a jhipster file (this was approved by Stephan). 

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Course with Postings enabled

1. Log in to Artemis as instructor
2. Navigate to course discussion or any exercise or lecture
3. Interact with postings (edit, create, delete, answer, react..) -> nothing unexpected should happen, no console error and not server error
4. Navigate as student and interact with postings (edit, create, delete, answer, react..) -> nothing unexpected should happen, no console error and not server error

### Review Progress





#### Code Review
#### Manual Tests
